readme.md

```diff
@@ -96,7 +96,8 @@ If you want to contribute, please read the [contribution guidelines](contributin
 
 - [Compat](https://github.com/amilajack/eslint-plugin-compat) - Lint browser compatibility of APIs used ([caniuse](http://caniuse.com/#search=fetch) as an ESLint plugin).
 - [ecmascript-compat](https://github.com/robatwilliams/es-compat) - Disable ECMAScript language features not supported by your browserslist targets.
-- [es](https://github.com/mysticatea/eslint-plugin-es) - Disable specific ECMAScript language versions or individual features.
+- ~~[es](https://github.com/mysticatea/eslint-plugin-es) - Disable specific ECMAScript language versions or individual features. No longer maintained.~~
```

Comment:
Since it's no longer maintained and the maintainer is not available, makes sense to cross out this line. WDYT?

Reply:
Any reason to keep the old es plugin listed? I'm cool with having only the new maintained es-x plugin listed.

Same with the other pull requests. I don't see any reason to list non-maintained/no longer working plugins.

Reply:
Updated.